### ISSUE TITLE
Alternative PinInput component library

### ISSUE THREAD

**Issue by rodw1995** [xamous/react-native-smooth-pincode-input#56]
Due to multiple issues with this library (and because it's not really maintained anymore?), I created a new PinInput component library, inspired on this one with a slightly different api. Maybe you can put a link in the readme to my new component library because it can be very useful for people looking for an alternative. The link is: https://github.com/rodw1995/react-native-pincode-input-component

**Comment by retyui** [comment]
@rodw1995 

Good idea and you can also use better an implementation:  [https://github.com/retyui/react-native-confirmation-code-field](https://bit.ly/2xfsRem)

|   |   |
|---|---|
| [![react-native-confirmation-code-field animated example](https://media.giphy.com/media/huJrqF0YRrNJBTwUmz/giphy.gif)](examples/DemoCodeField/src/AnimatedExample) | [![react-native-confirmation-code-field mask example](https://media.giphy.com/media/L4HHvT9Rwdlcdj59np/giphy.gif)](examples/DemoCodeField/src/MaskExample) [![react-native-confirmation-code-field underline example](https://media.giphy.com/media/XEazF64IwELNV8wZge/giphy.gif)](examples/DemoCodeField/src/UnderlineExample) [![react-native-confirmation-code-field basic example](https://media.giphy.com/media/WT4NHiw3XM0GWbjstc/giphy.gif)](examples/DemoCodeField/src/BasicExample) |

**Comment by rodw1995** [comment]
@retyui Looks good 👍

**Comment by bittu** [comment]
Fork of this with typescript [rn-pin-entry](https://github.com/bittu/rn-pin-entry)
